tests/test_clientsdaily.py

```diff
@@ -0,0 +1,69 @@
+import datetime as DT
+import pytest
+import json
+import os
+from mozetl.schemas import MAIN_SUMMARY_SCHEMA
+
+EXPECTED_INTEGER_VALUES = {
+    'active_addons_count_mean': 3613,
+    'crashes_detected_content_sum': 9,
+    'first_paint_mean': 12802105,
+    'pings_aggregated_by_this_row': 1122,
+    'search_count_sum': 1043
+}
+
+
+@pytest.fixture
+def make_frame(spark):
+    root = os.path.dirname(__file__)
+    path = os.path.join(root, 'resources',
+                        'main_summary-late-may-1123-rows-anonymized.json')
+    with open(path) as f:
+        jrows = json.load(f)
+        jrdd = spark.sparkContext.parallelize(map(json.dumps, jrows))
+        frame = spark.read.json(jrdd, MAIN_SUMMARY_SCHEMA)
```

Comment:
I'm wondering if `read.json` accepts the json file directly ([this example](https://spark.apache.org/docs/1.6.1/sql-programming-guide.html#creating-dataframes) reads from json, but doesn't show the shape of the data). Not an issue blocking this review.